README.md

```diff
@@ -106,7 +106,8 @@ The HLASM Language Support extension looks for locally stored members when a mac
 
 - `proc_grps.json` defines _processor groups_ by assigning a group name to a list of directories. Hence, the group name serves as a unique identifier of a set of HLASM libraries that are defined by a list of directories (some of which can be optional). Additionally, some assembler options can be specified in `asm_options` sections (`SYSPARM`, `SYSTEM_ID` and others).
 
-- `pgm_conf.json` provides a mapping between programs (open-code files) and processor groups. It specifies which list of directories is used with which program. 
+- `pgm_conf.json` provides a mapping between programs (open-code files) and processor groups. It specifies which list of directories is used with which program.
```

Comment:
`pgm_conf.json` provides the mapping, however `proc_grps.json` specifies the list of directories.

Reply:
```suggestion
- `pgm_conf.json` provides a mapping between programs (open-code files) and processor groups.
```